docs/events.yaml

```diff
@@ -658,6 +658,7 @@ x-events:
   event_type: host
   severity: "error"
   properties:
+    cluster_id: UUID_PTR
```

Comment:
why UUID_PTR and not just UUID ? 
The result will be the same, but according to the calling methods, we pass one cluster-id as a reference (clusterID), and one as its value (*clusterID):

```
eventgen.SendRefreshHostOrClusterStatusesFailedEvent(ctx, b.eventsHandler, clusterID, *hostID, *clusterID, err.Error())
```
unless there is a future plan to replace the first clusterID param with infraEnvID

Reply:
replaced clusterID with infraEnvID

Reply:
General comment about the uniformity of the events, for host-event it will be more acceptable to have the mandatory parameters first.
meaning, the first two params should be host-id and infraenv-id, later-on the cluster-id (when needed) and then the rest of the params. WDYT ?

Reply:
changed for the events related to this PR, I will create a separate PR for changing the other host-type events

Reply:
Agreed. Please file a Jira ticket for this. Otherwise, PR looks good.

Reply:
https://issues.redhat.com/browse/MGMT-8188